packages/chain-agnostic-permission/src/adapters/caip-permission-adapter-accounts.test.ts

```diff
@@ -183,4 +186,148 @@ describe('CAIP-25 eth_accounts adapters', () => {
       expect(input).not.toStrictEqual(result);
     });
   });
+
+  describe('setPermittedAccounts', () => {
```

Comment:
is it okay if we add a scenario to setting accounts where there is no scope defined in the caveat?